### DISCUSSION TITLE
代码中行高亮

### DISCUSSION THREAD

**Discussion by xiaohun254700** [hexojs/hexo#5130] [Question / Answer]
在编写技术博客，有时候代码只有少量的更改，把全部的代码粘贴出来，会让人不会注意到代码更改的地方，有没有一款插件能够让代码中的某一行高亮。

**Comment by renbaoshuo** [comment]
https://hexo.io/zh-cn/docs/tag-plugins#%E4%BB%A3%E7%A0%81%E5%9D%97

可以使用 `mark` 选项来对某一行进行高亮。

效果可以参考 https://oi.baoshuo.ren/libreoj-6009/#%E6%80%9D%E8%B7%AF 

![Screenshot_20221213_134028.png](https://arina.loli.net/2022/12/13/4fj1DIoOYNEHmyZ.png)

**Reply by xiaohun254700** [reply]
你好 看过你的博客 我这有很多算法的问题想咨询一下您 请问可以留一个联系方式吗
类似于上学的时候问您数学题的解题思路

**Reply by renbaoshuo** [reply]
> 你好 看过你的博客 我这有很多算法的问题想咨询一下您 请问可以留一个联系方式吗  
> 类似于上学的时候问您数学题的解题思路 

非常抱歉，我只是一名高中生，才疏学浅，可能并不能帮到您什么。另外，本人学业繁忙，因此并没有什么时间来为他人解答问题，还请您见谅。

**Reply by xiaohun254700** [reply]
再次感谢您解决了我的问题

**Comment by xiaohun254700** [comment]
我成功了 非常感谢
